apis/projectcontour/v1/httpproxy.go

```diff
@@ -294,6 +296,25 @@ type LoadBalancerPolicy struct {
 	Strategy string `json:"strategy,omitempty"`
 }
 
+// HeadersPolicy defines how headers are managed during forwarding
+type HeadersPolicy struct {
+	// Set defines how headers will be configured
```

Comment:
Let's document the semantics of `Set` here? Maybe something like this ...

```
Set specifies a list of HTTP header values that will be set in the HTTP header. If the corresponding HTTP header name already exists in the HTTP message, it will be replaced, otherwise a new HTTP header will be added.
```

Reply:
I'll update the comment to be more clear. Also, I want to be careful that we don't leak too much logic in the API spec.